runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h

```diff
@@ -328,6 +351,14 @@ struct AMDAIEDeviceModel {
   uint8_t getStreamSwitchArbiterMax(uint8_t col, uint8_t row) const;
   uint8_t getStreamSwitchMSelMax(uint8_t col, uint8_t row) const;
 
+  uint32_t getVectorLoadStoreAlignmentBits() const {
```

Comment:
I strongly dislike the whole OOP getters pattern thing - that's why this object is a struct and not a class in the first place - so that no one could make anything private/protected in the future (which these aren't btw so why would I use these getters ever ...). What you want to do is make these `const` so no one can change them